src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx

```diff
@@ -162,6 +162,13 @@ export default class AlterationEnrichmentContainer extends React.Component<IAlte
         return [];
     }
 
+    @computed get scatterPlotLabels() {
+        if(this.props.groups.length === 2) {
+            return [this.props.groups[0].name, this.props.groups[1].name];
+        }
+        return [];
```

Comment:
should this throw an error because it should not be called if there are more than two groups?

Reply:
those plots are not show if its more than 2

Reply:
right so what i'm saying is rather than return empty, we could throw an error in this situation.  not super important, but just a thought.  i mean, empty array seems wrong. though i guess typescript would complain if it didn't return anything.

Reply:
this is not important, just thinking about best practice